dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -603,9 +657,8 @@ def test_profile(self):
             quantiles = profile.pop('quantiles')
             expected_quantiles = expected_profile.pop('quantiles')
             actual_quartiles = {0: quantiles[249], 1: quantiles[499], 2: quantiles[749]}
-
-            self.assertDictEqual(expected_profile, profile)
-            self.assertEqual(expected_profile['precision'], 1.0)
+            self.assertCountEqual(expected_profile, profile)
+            self.assertCountEqual(expected_profile['precision'], profile['precision'])
```

Comment:
I don't believe we actually need the second assert, unless we want to be specific, but it needs to be `assertDictEqual`. This test should fail bc the current code has an error, but bc this is `assertCountEqual` it is passing.

Reply:
Also, `assertDictEqual` is necessary bc it checks all the values in the profile that this test is no longer checking

Reply:
Assertcountequal checks all the values, but doesn't care about order